domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -834,6 +883,34 @@ constructor(
             getResolutionSelector(sessionSettings.aspectRatio)
         )
 
+        if (currentSettings.value?.lowLightBoost == LowLightBoost.ENABLED) {
```

Comment:
Rather than reading this from `currentSettings`, `lowLightBoost` should be added to `TransientSessionSettings` and pulled out of settings in the the `currentSettings` Flow in `runCamera()`. That way we know we have a valid snapshot whenever we rebind the camera.